### PR TITLE
#4908 [Manager] Shabondi 連接到一個 Topic 後，Edit Properties 的 target topic…

### PR DESCRIPTION
**Pull request by konekoya**
… 沒有選到正確的值

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4908)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/api/apiInterface/connectorInterface.ts [hunk 1]
```diff
@@ -21,13 +21,15 @@ export enum SOURCES {
   ftp = 'oharastream.ohara.connector.ftp.FtpSource',
   smb = 'oharastream.ohara.connector.smb.SmbSource',
   perf = 'oharastream.ohara.connector.perf.PerfSource',
+  shabondi = 'oharastream.ohara.shabondi.ShabondiSource',
```
Comment (saivirtue) [inline ohara-manager/client/src/api/apiInterface/connectorInterface.ts:24]: Please help me remove the same key in `shabondiApi` :)

File: ohara-manager/client/src/components/Pipeline/PipelineUtils.js [hunk 2]
```diff
@@ -23,3 +25,11 @@ export const getPipelineOnlyTopicDisplayNames = topicCells => {
   if (topicIndex.length === 0) return 'T1';
   return `T${Number(topicIndex.pop()) + 1}`;
 };
+
+export const isShabondi = className =>
```
Comment (saivirtue) [inline ohara-manager/client/src/components/Pipeline/PipelineUtils.js:29]: 👍

File: ohara-manager/client/src/api/apiInterface/connectorInterface.ts (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/api/apiInterface/connectorInterface.ts:24]: No problem 🙆‍♂️

Comment (konekoya) [reply ohara-manager/client/src/api/apiInterface/connectorInterface.ts:24]: Done

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry

**Comment by konekoya** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by jackyoh** [comment]
retry

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**